### PR TITLE
[sd-2344]-(publish)Distinguishable confirmation for publish and kill …

### PR DESCRIPTION
**Pull request by sjunaid**
…(improvement)

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -448,11 +448,11 @@
         /**
          * In case $scope is dirty ask user if he want's to save changes and publish.
          */
-        this.confirmPublish = function confirmPublish() {
+        this.confirmPublish = function confirmPublish(action) {
             return modal.confirm(
-                gettext('There are some unsaved changes, do you want to save it and publish now?'),
+                gettext('There are some unsaved changes, do you want to save it and ' + action + ' now?'),
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/authoring.js]: this won't be picked by gettext, but you can use something like:
```
$interpolate(gettext('There are ... {{ action }} now?'))({action: action});
```

Comment (sjunaid) [inline client/app/scripts/superdesk-authoring/authoring.js]: nice

**Pull request merged**